.github/workflows/build.yaml

```diff
@@ -70,19 +70,34 @@ jobs:
           MACOSX_DEPLOYMENT_TARGET: 10.14
         run: |
           cargo build --verbose --release
-          mv target/release/eslauncher2 eslauncher2-x86_64-apple-darwin
+          chmod 755 target/release/eslauncher2
+          ls -al target/release
+      - name: Package binary
+        run: |
+          cargo install cargo-bundle
+          cargo bundle --release
+          ls -al target/release/bundle/osx/ESLauncher2.app/Contents/MacOS
```

Comment:
should get rid of this `ls` before merge.

Reply:
I would like to keep that one in, reports only on the executable.